app/Http/Controllers/Admin/MonsterCrudController.php

```diff
@@ -240,76 +337,97 @@ public function setupShowOperation()
             'type'          => 'model_function_attribute',
             'function_name' => 'getCategory', // the method in your Model
             'attribute'     => 'name',
+            'tab'           => 'Function/Closure',
         ]);
 
         $this->crud->addColumn([
             'name'  => 'number', // The db column name
             'key'   => 'phone',
             'label' => 'Phone', // Table column heading
             'type'  => 'phone',
+            'tab'   => 'Simple',
         ]);
 
         $this->crud->addColumn([   // Upload
             'name'   => 'upload_multiple',
             'label'  => 'Upload Multiple',
             'type'   => 'upload_multiple',
             'disk'   => 'public',
+            'tab'    => 'Uploads',
+        ]);
+
+        $this->crud->addColumn([
+            'name'  => 'switch',
+            'label' => 'Switch',
+            'type'  => 'switch',
+            'tab'   => 'Simple',
         ]);
 
         $this->crud->addColumn([
             'name'  => 'switch',
-            'label' => 'Checkbox',
-            'type'  => 'checkbox',
+            'key'   => 'check',
+            'label' => 'Check',
+            'type'  => 'check',
+            'tab'   => 'Simple',
         ]);
 
         $this->crud->addColumn([
             'name'  => 'my_custom_html',
             'label' => 'Custom HTML',
             'type'  => 'custom_html',
             'value' => '<span class="text-danger">Something</span>',
+            'tab'   => 'Miscellaneous',
+        ]);
+
+        $this->crud->addColumn([
+            'name'  => 'view',
+            'label' => 'Custom View',
+            'type'  => 'view',
+            'view'  => 'crud::columns.something',
+            'tab'   => 'Miscellaneous',
         ]);
 
         $this->crud->addColumn([
             'name'  => 'features',
             'label' => 'JSON',
             'type'  => 'json',
+            'tab'   => 'Miscellaneous',
         ]);
 
         $this->crud->addColumn([
             'name'      => 'id',
             'type'      => 'number',
             'label'     => '#',
             'orderable' => false,
+            'tab'       => 'Simple',
         ]);
 
         $this->crud->addColumn([
             'name'  => 'browse_multiple',
             'type'  => 'array',
```

Comment:
we have the browse_multiple column right ? Why not use it ?

Reply:
Changed to `browse_multiple